myhpi/core/models.py

```diff
@@ -152,11 +152,16 @@ class UserSelectWidget(s2forms.ModelSelect2MultipleWidget):
 
 class Minutes(BasePage):
     date = DateField()
-    moderator = ForeignKey(User, on_delete=models.PROTECT, related_name="moderator")
-    author = ForeignKey(User, on_delete=models.PROTECT, related_name="author")
+    moderator = ForeignKey(
+        User, blank=True, null=True, on_delete=models.PROTECT, related_name="moderator"
+    )
+    author = ForeignKey(
+        User, blank=True, null=True, on_delete=models.PROTECT, related_name="author"
+    )
     participants = ParentalManyToManyField(User, related_name="minutes")
     labels = ClusterTaggableManager(through=TaggedMinutes, blank=True)
     text = CustomMarkdownField()
+    guests = models.JSONField(blank=True, default=[])
```

Comment:
JSON feels a bit hacky to me at this point. Maybe we want to approach this similar to abbreviation explanations?

Reply:
I want to be able to add guests on the same page and not need to create a guest first in some other menu and then select that on the minutes page

Reply:
We want to store a list of names, with no additional information. JSON is a type used to store a list of strings, so I think in the db it is fine.

Reply:
all right, makes sense